modules/gdscript/gdscript_warning.cpp

```diff
@@ -162,10 +153,16 @@ String GDScriptWarning::get_message() const {
 			return vformat(R"*(The default value is using "%s" which won't return nodes in the scene tree before "_ready()" is called. Use the "@onready" annotation to solve this.)*", symbols[0]);
 		case ONREADY_WITH_EXPORT:
 			return R"("@onready" will set the default value after "@export" takes effect and will override it.)";
+#ifndef DISABLE_DEPRECATED
+		case PROPERTY_USED_AS_FUNCTION:
+		case CONSTANT_USED_AS_FUNCTION:
+		case FUNCTION_USED_AS_PROPERTY:
+			break; // Never produced.
+#endif
```

Comment:
I don't think it's a good idea deprecating these just because they're unused. They should be produced instead, when relevant.

`FUNCTION_USED_AS_PROPERTY` is okay, since we have Callable now, but the others could be helpful.

Reply:
```suggestion
#ifndef DISABLE_DEPRECATED
		case PROPERTY_USED_AS_FUNCTION:
		case CONSTANT_USED_AS_FUNCTION:
		case FUNCTION_USED_AS_PROPERTY:
			break; // Warnings never produced as they already produce errors.
#endif
```